acceptance-tests.md

```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate). Note the identities for the given user, specifically one that starts with `github:`, like `github:15114160`.
+        2. For a given users identity, `github:15114160` for example, [Click here to check that the given identity is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~Identity&name=github%3A15114160).
 2. **Remove a user**
     * _As an administrator, I should be able to remove a user, labels association with the user, and user identity from the cluster._
        * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A NERC admin uses a private runbook to disable/ deactivate a user from KeyCloak.
         2. _(Optional)_ After that is done, the NERC admin uses the OpenShift CLI and runs the following:
             1. `oc delete user &lt;username>`
             2. `oc delete identity &lt;identity_provider>:&lt;username>`
         3. The user can no longer access OpenShift with their credentials
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is no longer listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate).
+        2. For a given users identity, `github:15114160` for example, [Click here to check that the given identity is no longer listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~Identity&name=github%3A15114160).
 3. **Add/Remove PI privilege to a user**
     * _For any user account, the administrator should be able to add or remove PI status associated with that account.  A user may be a PI on multiple projects, but a project can have only 1 PI._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. User creates a new account as described in section 1.
         2. User fills out the PI request form.
         3. To approve the request, NERC admin assigns the user to the PI role on KeyCloak’s user management.
         4. NERC admin approves the request and responds with a ticket reply.
         5. To remove the PI role, just reverse the step iii. And send out an email to the user informing about it.
+    * Acceptance tests: requires investigation
```

Comment:
What investigation is happening?

Reply:
add/remove admin priv. is missing as an acceptance test

Reply:
Text has been revise and now includes appropriate steps.    Acceptance test step "e"  still needs edits, as it is the acceptance test for deleting PI role from a user, but the check doesn't show that.  Reference to step "iii" in earlier text should be step "c"

Reply:
@computate ^